packages/core/src/index.ts

```diff
@@ -201,6 +227,39 @@ export class Abby<
     return this.getProjectData();
   }
 
+  /**
+   * Helper function to retrieve the time a flag is valid
+   * @param key
+   * @returns
+   */
+  getFeatureFlagTimeout<F extends FlagName>(key: F) {
+    return this.#flagTimeoutMap.get(key);
+  }
+
+  /**
+   * Helper function to check if a featureflag should be refetched
+   * @param key name of the featureflag
+   * @returns value of flag
+   */
+  getValidFlag<F extends FlagName>(key: F, refetch: boolean | undefined) {
```

Comment:
this may break every call in every lib. if this is not intended, maybe do `refetch?: boolean` instead. :)